spec/lib/annotate/annotate_routes_spec.rb

```diff
@@ -272,14 +272,15 @@
   end
 
   describe 'As for Rake versions' do
-    context 'with older Rake versions' do
-      before(:each) do
-        expect(File).to receive(:exist?).with(ROUTE_FILE).and_return(true)
-        expect(AnnotateRoutes).to receive(:`).with('rake routes').and_return(rake_routes_result)
-        expect(File).to receive(:open).with(ROUTE_FILE, 'wb').and_yield(mock_file)
-        expect(AnnotateRoutes).to receive(:puts).with(MESSAGE_ANNOTATED)
-      end
+    before :each do
+      expect(File).to receive(:exist?).with(ROUTE_FILE).and_return(true)
+      expect(File).to receive(:open).with(ROUTE_FILE, 'wb').and_yield(mock_file)
+      expect(File).to receive(:read).with(ROUTE_FILE).and_return(route_file_content)
+
+      expect(AnnotateRoutes).to receive(:`).with('rake routes').and_return(rake_routes_result)
```

Comment:
This is interesting 🤔. I had no idea that when you do 
```
`rake routes`
```

It sends the message with method name `. Very neat.